docs/cli/dnx-migration.md

```diff
@@ -0,0 +1,155 @@
+# Migrating from DNX to .NET Core CLI
+
+## Overview
+With RC1 release of .NET Core and ASP.NET Core 1.0, we introduced DNX tooling to the world. With RC2 release of .NET 
+Core and ASP.NET Core 1.0 we are pivoting on using the .NET Core CLI.
```

Comment:
Don't say "pivoting" here :). Just say "we are transitioning to use the .NET Core CLI"

![](https://31.media.tumblr.com/49136fba255c5980f88be4971e4d29aa/tumblr_n6pf0jcbYU1r7uxhho6_400.gif)